internal/utils/utils.go

```diff
@@ -189,3 +190,8 @@ func StringToHash(s string) (*string, error) {
 	hash := fmt.Sprintf("%x", bs)
 	return &hash, nil
 }
+
+func FormatStrictDateTime(t time.Time) string {
+	strictDateTime := fmt.Sprintf("%d-%02d-%02dT%02d:%02d:%02d.%03dZ", t.Year(), t.Month(), t.Day(), t.Hour(), t.Minute(), t.Second(), t.Nanosecond())
```

Comment:
Should be able to replace this with `.Format`

```suggestion
	strictDateTime := t.Format("2006-01-02T15:04:05.000Z")
```

Reply:
This certainly makes life a little easier! 😆 Will update this

Reply:
done 👍